src/sentry/apidocs/examples/release_threshold_examples.py

```diff
@@ -0,0 +1,33 @@
+from drf_spectacular.utils import OpenApiExample
+
+from .project_examples import BASE_PROJECT
+
+
+class ReleaseThresholdExamples:
+    THRESHOLD_STATUS_RESPONSE = [
+        OpenApiExample(
+            "Client key with rate limiting",
+            value={
+                f"{BASE_PROJECT['slug']}-v1.0.0": [
+                    {
+                        "project_id": BASE_PROJECT["id"],
```

Comment:
"id" here is actually a string in `BASE_PROJECT`, it's prob better to just write out a number